enclave_build/src/lib.rs

```diff
@@ -240,6 +240,7 @@ impl<'a> Docker2Eif<'a> {
         let output = Command::new(&self.linuxkit_path)
             .args(&[
                 "build",
+                "-docker",
```

Comment:
New flag was added to first get the image from docker cache before linuxkit cache